microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,258 @@
+## Start by doing automated testing against Edge Canary
+
+Start with automated testing against the Microsoft Edge **Canary** channel, which ships daily.  This enables you to catch issues as early as possible. The Microsoft Edge **Dev** channel ships weekly, and is also a good option.
+
+Do the following steps:
+1. Set the preview channel for your app.  See [Ways to set the preview channel](#ways-to-set-the-preview-channel), below.
+1. Deploy the non-stable channels.  See [How to deploy non-stable channels](#how-to-deploy-non-stable-channels), below.
+1. Conduct automated testing on your app running against the non-stable channels.  See [How to do automated testing](#how-to-do-automated-testing), below.
+1. Compare the results with the baseline.  See [The baseline release of WebView2 for testing](#the-baseline-release-of-webview2-for-testing), below.
+
+Upon finding issues, you can report them via the [WebView2 feedback repo](https://github.com/MicrosoftEdge/WebView2Feedback).
+
+
+<!-- ====================================================================== -->
+## Ways to set the preview channel
+
+You can set the preview channel by doing either of the following:
+* Use `ChannelSearchKind` to set the channel search order.
+* Use the browser executable folder.
+
+These options are explained below.
+
+
+<!-- ====================================================================== -->
+## Setting the preview channel by using ChannelSearchKind to set the channel search order
+
+The channel search kind controls the default channel search order.  By default, the WebView2 loader searches for binaries from most-stable to least-stable (Canary), using the first binary that's found:
+
+```
+WebView2 Runtime (Stable) ->  Edge Beta -> Edge Dev -> Edge Canary
+```
+
+To do prerelease testing, reverse the channel search order, so that the loader looks for binaries in order from least-stable (Canary) to most-stable:
+
+```
+Edge Canary -> Edge Dev -> Edge Beta -> WebView2 Runtime (Stable)
+```
+
+The channel search kind can be set per app through a registry key, environment variable, API, or group policy.
+
+The value `1` indicates reversed search order; that is, from least-stable (Edge Canary) to most-stable.
+
+
+##### [Registry key](#tab/registry-key)
+
+```reg
+REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ChannelSearchKind /v WebView2APISample.exe /t REG_DWORD /d 1
+```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.
+
+
+##### [Environment variable](#tab/environment-variable)
+
+Name: `WEBVIEW2_CHANNEL_SEARCH_KIND`  
+Value: `1`
+
+
+##### [API](#tab/api)
+
+By default, the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property is `CoreWebView2ChannelSearchKind.MostStable` (an enum value).  Instead, reverse the search order by setting the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property to `CoreWebView2ChannelSearchKind.LeastStable`.
+
+.NET:
```

Comment:
nit: I have seen .NET/WinRT/Win32 showing in the tabs - not in the list. Hope it is possible here also - but I'm not sure

Reply:
to research nested tas:
https://review.learn.microsoft.com/en-us/help/platform/tabbed-conceptual-overview?branch=main&tabs=windows%2Cazure-cli#limitations-of-tabs

Reply:
too much research required, for too little gain.  keeping as-is.  rendered section:
https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3053&tabs=api#set-the-preview-channel-by-using-channelsearchkind